ofl/solway/DESCRIPTION.en_us.html

```diff
@@ -1,3 +1,4 @@
-<p>Solway is a playful slab-serif with a charming and understated tone of voice. Created by The Northern Block’s Mariya Pigoulevskaya, the font is inspired by the warmth of community and the idyll of rural-living. It will find perfect expression within lifestyle brands, nature blogs and cook-books.</p>
-
-<p>To contribute, see <a href="https://github.com/TheNorthernBlock/solway">github.com/TheNorthernBlock/solway</a></p>
+<p>Solway is a playful slab-serif with a charming and understated tone of voice, inspired by the warmth of community and the idyll of rural-living.</p>
+<p>It is a contemporary family defined by the soft corners, ball terminals, rounded slabs that give it a welcoming and friendly appeal. Its monoline structure and minimal stroke contrast aspire to cohesion and stability.</p>
+<p>Available in five weights, it is suitable for a variety of projects. It could find perfect expression within lifestyle brands or nature, travel and foodie blogs.</p>
+<p>Designed by <a href="http://www.metype.co.uk" target="_blank">Mariya V. Pigoulevskaya</a> for <a href="https://thenorthernblock.co.uk" target="_blank">The Northern Block</a>. To colaborate, see <a href="https://github.com/mashavp/Solway" target="_blank">Solway GitHub</a>.</p>
```

Comment:
`colaborate` typo, should be `collaborate`

Reply:
It should be `contribute` actually!
How can I fix it? Should I change it on the repo and restart the PR or it should be another way, given it's already merged.

Reply:
Just fix it and run the dispatcher again.

Reply:
Ok, thanks! Fixed on [https://github.com/google/fonts/pull/2252](https://github.com/google/fonts/pull/2252)